pkg/synopsysctl/cmd_delete.go

```diff
@@ -132,6 +132,28 @@ var deleteOpsSightCmd = &cobra.Command{
 	},
 }
 
+// deletePolarisCmd deletes Polaris instances from the cluster
+var deletePolarisCmd = &cobra.Command{
+	Use:           "polaris NAME...",
+	Example:       "synopsysctl delete polaris <name>\nsynopsysctl delete polaris <name1> <name2> <name3>",
+	Short:         "Delete one or many polaris instances",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 0 {
+			cmd.Help()
+			return fmt.Errorf("this command takes no argument")
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if err := kubeClient.CoreV1().Secrets(namespace).Delete("polaris", &metav1.DeleteOptions{}); err != nil {
```

Comment:
This is hard coded. What if the user enters a different name or multiple instances.

Reply:
Synopsysctl doesn't take a name anymore since we only support one instance per namespace and that we no longer use custom resources. Therefore, the secret name will always be the same. The user can still configure the namespace with --namespace